draft-ietf-quic-transport.md

```diff
@@ -1257,6 +1270,27 @@ frames), the receiving peer MAY send an ACK frame after a reasonable number
 Strategies and implications of the frequency of generating acknowledgments are
 discussed in more detail in {{QUIC-RECOVERY}}.
 
+## Special Considerations for Path MTU Discovery 
+
+Traditional ICMP-based path MTU discovery ({{!RFC1191}}, {{!RFC1981}}) is
+potentially vulnerable to off-path attacks that succesfully guess the IP/port
+4-tuple and reduce the MTU to a bandwidth-inefficient value. TCP connections
+mitigate this risk by using the (at minimum) 8 bytes of transport header
+echoed in the ICMP message to validate the TCP sequence number as valid for
+the current connection. However, as QUIC operates over UDP, the echoed
+information may consist only of the IP and UDP headers, which usually has
+insufficient entropy to mitigate off-path attacks.
+
+As a result, endpoints that implement PMTUD SHOULD take steps to mitigate this
+risk, which may include:
```

Comment:
nit: "SHOULD take steps to mitigate this risk. For instance, an application may:"